### DISCUSSION TITLE
Sharing: Main navigation bar for shared links

### DISCUSSION THREAD

**Discussion by CWempe** [photoprism/photoprism#3948] [Ideas]
**Is your feature request related to a problem? Please describe.**

I want to share one or more albums with a bunch of people. (guest of a wedding)
While I can [share multiple albums with one link](https://github.com/photoprism/photoprism/issues/1330#issuecomment-1024883565), the visitors cannot use the great features like people, places, labels, calendar, events from the main navigation bar.
These are the features that separate PhotoPrism from other file sharing services like NextCloud etc.

![image](https://user-images.githubusercontent.com/1270359/177417675-394b7aab-71f9-4818-8952-10e3dea5275b.png)

**Describe the solution you'd like**

Make it possible to enable these features for public users.
Add some checkboxes to the share dialog.

![image](https://user-images.githubusercontent.com/1270359/177417862-a8eb4eb3-7971-4990-8baf-d142683f4c13.png)

But as a first step enabling `all` or `none` would be enough.

**Describe alternatives you've considered**

I do not know any....

**Additional context**

This would also be a very good advertisement, because many new people would see the cool features of PhotoPrism. 😎

**Comment by graciousgrey** [comment]
Somehow related to : https://github.com/photoprism/photoprism/issues/557.

**Comment by graciousgrey** [comment]
Not sure if this is a suitable use case for sharing links. Might get really complicated  e.g. if you select the "people" option for one album you share with a link, but not for another album that is shared with the same link.   Also with the search people can search with specific search filters and reveal metadata you might not wanted to share... 

Might be better to solve this with read-only users.

**Comment by CWempe** [comment]
You are right, if I select `people` for one shared album it should have the same settings as any other shared album with the same link. I understand this might get confusing for the user.

> Also with the search people can search with specific search filters and reveal metadata you might not wanted to share...

I thought this was already possible, because when I opened the shared link I could see other albums like events and places.
But that seems to be an error, because I was logged in with the same browser, so while opening the shared link I had the admin privileges.

> Might be better to solve this with read-only users.
In my mind each shared link (cryptic url) is like a user.
When I use the same link for another album I "add the album to a user". 😅

But I understand this is a complicated topic.
